services/core-api/app/api/projects/project/resources/project.py

```diff
@@ -191,7 +188,9 @@ def get(self):
                         'project_lead_party_guid': project.project_lead_party_guid,
                         'project_lead_name': project.project_lead_name,
                         'update_timestamp': update_timestamp,
-                        'mine': project.mine
+                        'mine': {
+                            'mine_name': project.mine.mine_name
```

Comment:
Is this the only field we're using here out of all the mine info :laughing:

Reply:
I've missed the `Commodity`  field, which is also retrieved from mine object. I just pushed the fix for that.

Table Headers => `Project Title`, `Project ID`, `Mine name`, `MRC, Stage`, `Review Status`, `EMLI Project Lead`, `Commodity`, `Updated Date`

Attached image shows the updated response, with the relevant fields for the table view.
![Updated Response](https://user-images.githubusercontent.com/118843449/231281481-1b67678f-3c5f-456e-9d14-104696997885.png)